src/explorer/model/Dependencies.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as vscode from "vscode";
+import { ITreeItem } from "./ITreeItem";
+
+const DUPLICATE_VALUE: string = "omitted for duplicate";
+const CONFLICT_VALUE: string = "omitted for conflict";
+
+export class Dependencies implements ITreeItem {
+    public dependency: string;
+    public pomPath: string;
+    private curDep: string;
+    private separator: string;
+    private collapsedState: vscode.TreeItemCollapsibleState;
```

Comment:
`vscode.TreeItemCollapsibleState` is property of TreeItem, which should be controlled by vscode.
If you want to specify whether a dependency is expandable (has children), it's better to define a boolean member e.g. `hasChildren/Dependencies`.